pkg/bundle/bundle.go

```diff
@@ -150,11 +140,18 @@ func Pack(directory string) (io.Reader, error) {
 
 	matcher := gitignore.NewMatcher(patterns)
 
-	var buf bytes.Buffer
```

Comment:
Rational: `bytes.Buffer` does not implement `ReadCloser`. Instead of wrapping a `NopReadCloser` around it, I figured that `os.Pipe()` with the `os.File` implement the interface we need.